src/main/webapp/app/shared/statistics-graph/statistics-graph.component.ts

```diff
@@ -60,26 +64,41 @@ export class StatisticsGraphComponent implements OnChanges {
         this.createLabels();
         this.chartData = [
             {
-                label: this.amountOfStudents,
+                label: this.labelTitle,
                 data: new Array(this.barChartLabels.length).fill(0),
                 backgroundColor: 'rgba(53,61,71,1)',
                 borderColor: 'rgba(53,61,71,1)',
                 hoverBackgroundColor: 'rgba(53,61,71,1)',
             },
         ];
         this.createCharts();
-        this.service.getChartData(this.currentSpan, this.currentPeriod, this.graphType).subscribe((res: number[]) => {
-            this.dataForSpanType = res;
-            this.chartData = [
-                {
-                    label: this.amountOfStudents,
-                    data: this.dataForSpanType,
-                    backgroundColor: 'rgba(53,61,71,1)',
-                    borderColor: 'rgba(53,61,71,1)',
-                    hoverBackgroundColor: 'rgba(53,61,71,1)',
-                },
-            ];
-        });
+        if (this.statisticsView === StatisticsView.ARTEMIS) {
+            this.service.getChartData(this.currentSpan, this.currentPeriod, this.graphType).subscribe((res: number[]) => {
+                this.dataForSpanType = res;
+                this.chartData = [
+                    {
+                        label: this.labelTitle,
+                        data: this.dataForSpanType,
+                        backgroundColor: 'rgba(53,61,71,1)',
+                        borderColor: 'rgba(53,61,71,1)',
+                        hoverBackgroundColor: 'rgba(53,61,71,1)',
```

Comment:
Putting this color into a variable might be better.

Reply:
👍